apps/remote_control/lib/lexical/remote_control/dispatch.ex

```diff
@@ -46,6 +48,7 @@ defmodule Lexical.RemoteControl.Dispatch do
     case :gen_event.start_link(name()) do
       {:ok, pid} = success ->
         Enum.each(@handlers, &:gen_event.add_handler(pid, &1, []))
+        register_progress_listener()
```

Comment:
One thing I don't love is having to patch `register_progress_listener` in so many tests. We're not using the `Dispatch.start_link` argument right now -- what if we added a `:progress` opt that defaults to `true` and only call `register_progress_listener` if it's set?

Then, in tests, where we don't want to register for progress, we can do:

```elixir
start_supervised!({Dispatch, progress: false})
```

Reply:
Good idea

Reply:
I think we make `progress` default to `nil`, and set `true` in the `application.ex`, because most tests or processes do not need to care about the progress.